CODEOWNERS

```
@@ -0,0 +1,27 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @Rocketknight1 # if no one is pinged based on the other rules, he will do the dispatch
+**.md @stevhliu
+docs/ @stevhliu
+/benchmark/ @McPatate
+/src/transformers/models/*/*_processing_** @molbap @yonigozlan
+/src/tranformers/models/*/*image_processing* @qubvel
+/src/transformers/models/*/*_image_processing_*_fast* @yonigozlan
```

If I correctly got the logic: 
 - for everything related to processing @molbap and @yonigozlan (added myself as well)
 - specifically for image processors @qubvel
 - specifically for fast image processors @yonigozlan 

+ fix matching patterns

```suggestion
/src/transformers/models/*/*processing* @molbap @yonigozlan @qubvel
/src/transformers/models/*/image_processing* @qubvel
/src/transformers/models/*/image_processing_*_fast* @yonigozlan
```